portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -1187,7 +1187,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-129395. View external option remember device."
+	@description = "This is a use case for LPS-129395. Remember device option should be visible in Oauth2 admin page if it was set in the application settings."
```

Comment:
Please use the official letters in descriptions: Oauth2 -» OAuth 2
cc @ferenconodiD